### PR TITLE
Add newsletter form to blog

### LINKED ISSUE #414: Build newsletter signup form
Add to blog index and blog view ala https://ubuntu.com/blog and https://ubuntu.com/blog/data-science-workflows-on-kubernetes-with-kubeflow-pipelines-part-2

[Brief](https://docs.google.com/document/d/1X7yaWha8Z2PbKsrvwX7nOAZhUwEqXCT2H9tVDSflJjM/edit)

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Add newsletter form to blog homepage and blog post page

## QA

- Go to /blog
- Check that there is a newsletter subscription form
- Check that the form works
- Go to a blog post page
- Check that there is a newsletter subscription form
- Check that the form works
- Check that the form copy matches [the copy doc](https://docs.google.com/document/d/12FTdNsmhTF6KPtYN5lQ8iFQgvU9BgJDdCvUjiScc-Ok/edit)

## Issue / Card

Fixes #414

### CODE REVIEW

**Comment by steverydz** [comment]
@zprood can you verify that the newsletter form is working your end? https://cn-ubuntu-com-418.demos.haus/blog

**Comment by zprood** [comment]
@steverydz Yes, it looks good. Just a small change that there's a space behind "同意的 隐私", can you help to remove the space ? Thanks. 
image

**Comment by steverydz** [comment]
@zprood no worries, done

**Comment by zprood** [comment]
I checked the form id on marketo, it's correct. It's weird.

File: templates/blog/newsletter-form.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        </span>
+
+        <input type="hidden" name="Consent_to_Processing__c" value="yes" />
+
+        <input value="1959" class="mktoField mktoFieldDescriptor" name="lpId" type="hidden">
+
+        <input value="30" class="mktoField mktoFieldDescriptor" name="subId" type="hidden">
+
+        <input name="lpurl" value="https://pages.canonical.com/Insights-Subscription_Insights-Subscription-test.html?cr={creative}&amp;kw={keyword}" type="hidden">
+        <input value="3647" class="mktoField mktoFieldDescriptor" name="formid" type="hidden">
```
Comment (pmahnke) [inline templates/blog/newsletter-form.html:70]: you need 'formVid' as well
```suggestion
        <input value="3647" class="mktoField mktoFieldDescriptor" name="formid" type="hidden">
                <input value="3647" class="mktoField mktoFieldDescriptor" name="formVid" type="hidden">
```

**Pull request merged**